sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/ui/ToolTips.scala

```diff
@@ -29,4 +29,18 @@ private[ui] object ToolTips {
 
   val THRIFT_SERVER_DURATION =
     "Difference between start time and close time"
+
+  val THRIFT_SESSION_TOTAL_EXECUTE =
+    "Number of operations submitted in this session."
+
+  val THRIFT_SESSION_START_TIME =
+    "Session Start time, on launching the session."
+
+  val THRIFT_SESSION_FINISH_TIME =
+    "Time that the session was closed."
+
+  val THRIFT_SESSION_DURATION =
+    "Total duration the session has been active, from the time session" +
+      " was lauched till it was closed."
```

Comment:
lauched till -> launched until
But should we say 'started' instead anyway, to be consistent with the above?